content/docs/1.3.0/advanced-resources/deploy/customizing-default-settings.md

```diff
@@ -31,7 +33,10 @@ From the project view in Rancher, go to **Apps > Launch > Longhorn** and edit th
     git clone https://github.com/longhorn/longhorn.git
     ```
 
-2. Modify the config map named `longhorn-default-setting` in the yaml file `longhorn/deploy/longhorn.yaml`. For example:
+2. Modify the config map named `longhorn-default-setting` in the yaml file `longhorn/deploy/longhorn.yaml`.
+
+   In the below example, users customize the default settings, backup-target, backup-target-credential-secret, and default-data-path.
+   When the setting is absent or has a leading `#` symbol, the default setting will use the default value in Longhorn or the customized values previously configured.
```

Comment:
Should we need to emphasize the setting should need to have a value instead of `empty`? If empty, what happened?

Reply:
If it is empty and valid, the value will be cleaned up. If it is not valid, longhorn-manager will ignore it.

Reply:
Thanks. Probably we can briefly mention that as well.

Reply:
Updated. Thank you.